pkg/document/crdt/tree_test.go

```diff
@@ -465,6 +465,26 @@ func TestTreeEdit(t *testing.T) {
 		assert.Equal(t, 0, idx)
 	})
 
+	t.Run("marshal test", func(t *testing.T) {
+		ctx := helper.TextChangeContext(helper.TestRoot())
+		tree := crdt.NewTree(crdt.NewTreeNode(helper.PosT(ctx), "root", nil), helper.TimeT(ctx))
+		_, err := tree.EditT(0, 0, []*crdt.TreeNode{crdt.NewTreeNode(helper.PosT(ctx), "p", nil)}, 0,
+			helper.TimeT(ctx), issueTimeTicket(ctx))
+		assert.NoError(t, err)
+		_, err = tree.EditT(1, 1, []*crdt.TreeNode{
+			crdt.NewTreeNode(helper.PosT(ctx), "text", nil, "\"Hello\" \n i'm yorkie!"),
```

Comment:
It would be good to use `raw string literals` for readability.

```suggestion
			crdt.NewTreeNode(helper.PosT(ctx), "text", nil, `"Hello" \n i'm yorkie!`),
```

Please refer to this commit: https://github.com/yorkie-team/yorkie/commit/e590dad23ba2296b4bda9eefd30d8416882ae479